src/app/layout.js

```diff
@@ -17,10 +18,12 @@ export default function RootLayout({ children }) {
   return (
     <html lang="ua">
       <body className={cn(montserrat.className, 'relative scroll-smooth')}>
-        <QueryContext>
-          <main className="bg-other-white">{children}</main>
-        </QueryContext>
-        <div id="modal-root" />
+        <Hint>
```

Comment:
I already saw Hint component wrapping the content in src/app/(app)/layout.js, do we need it in both places? And why do we need global provider for this? We could use locally only where we need the hint

Reply:
Thanks for comment! Honestly, we need just one, fixed it.